connexion/utils.py

```diff
@@ -148,12 +148,20 @@ def loads(self, data):
 def has_coroutine(function, api=None):
```

Comment:
Since the logic of this function is getting more complex can you add a docstring explaining it?

Reply:
I will do that @jmcs

Reply:
hey @dutradda , any chance you can pick this back up?